### DISCUSSION TITLE
How to overcome merge GraphQL input type error

### DISCUSSION THREAD

**Discussion by GauravAgarwal009** [ever-co/ever#1556] [Q&A]
I am unable to merge the GraphQL input type error.
![Screenshot 2023-03-24 at 1 54 22 PM](https://user-images.githubusercontent.com/65541324/227465874-131b4907-85d5-4323-a4f9-cacdadde41dc.png)

**Comment by evereq** [comment]
@GauravAgarwal009 can you provide more context of what you try to do? What commands you run etc?
